recipes/lodei/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: lodei
+  version: 1.0.0
```

Comment:
Please use jinja variables for the version and reuse them in the URL. This will enable our bot to update the package when you release a new version